pkg/common/cns-lib/vsphere/utils.go

```diff
@@ -391,3 +395,37 @@ func isVsan67u3Release(ctx context.Context, m *defaultVirtualCenterManager, host
 	log.Debugf("vCenter version is :%q", vc.Client.Version)
 	return vc.Client.Version == cns.ReleaseVSAN67u3, nil
 }
+
+// IsvSphereVersion70U3orAbove checks if specified version is 7.0 Update 3 or higher
+// The method takes aboutInfo{} as input which contains details about
+// VC version, build number and so on.
+// If the version is 7.0 Update 3 or higher, the method returns true, else returns false
+// along with appropriate errors during failure cases
+func IsvSphereVersion70U3orAbove(ctx context.Context, aboutInfo types.AboutInfo) (bool, error) {
```

Comment:
Normally we have some vsphere version checks in govmomi. Any reason we dont have this check there?

Reply:
vSphere versions in gvmomi contain only the released VSAN version numbers. During development ,VSAN versions are not same as its released versions.